openquake/hazardlib/contexts.py

```diff
@@ -727,9 +730,8 @@ def _get_ctx_planar(self, mag, planar, sites, src_id, start_stop, tom):
         # setting distance parameters
         for par in dists:
             ctx[par] = dists[par]
```

Comment:
Here you are introducing a performance penalty in ALL calculations were `few_sites` is false that should be measured.

Reply:
If modified to only set distance params when self.few_sites, test case 66 in classical calculator tests fails (in this test it is checked that you can specify both a site and a site model and the engine will automatically get the closest site model parameters).

Reply:
Now only if self.fewsites are closest points coordinates assigned